targets/frontend/package.json

```diff
@@ -13,30 +13,39 @@
     "@sentry/integrations": "^5.23.0",
     "@sentry/node": "^5.23.0",
     "@shared/graphql-client": "^1.0.0",
+    "@shared/id-generator": "^1.0.0",
     "@socialgouv/cdtn-slugify": "^4.30.0",
     "@socialgouv/cdtn-sources": "^4.30.0",
     "@socialgouv/matomo-next": "^1.1.2",
+    "@socialgouv/react-ui": "^4.17.0",
     "@zeit/next-source-maps": "0.0.4-canary.1",
     "ace-builds": "^1.4.12",
     "argon2": "^0.27.0",
+    "cookie": "^0.4.1",
+    "d3": "^6.1.1",
+    "d3-hierarchy": "^2.0.0",
     "diff": "^4.0.2",
     "graphql": "^15.3.0",
     "http-proxy-middleware": "^1.0.5",
     "isomorphic-unfetch": "^3.0.0",
     "jsonwebtoken": "^8.5.1",
+    "memoizee": "^0.4.14",
```

Comment:
this import is not used, we can remove it

Reply:
Is there a tool you use to detect unused packages ?

Reply:
yes, indeed,  it's called memory :D

Reply:
Arf, mine never really worked :/